apps/dog-walking-app/app/globals.css

```
@@ -0,0 +1,58 @@
+@tailwind base;
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)